### PR TITLE
DXCDT-386: Expand logs test cases

### PR DESCRIPTION
**Pull request by sergiught**
### 🔧 Changes

Expands the test cases for the logs command and its relative subcommands and also fixes raised bugs. More in the comments down below. 

### 📝 Checklist

- [x] All new/changed/fixed functionality is covered by tests (or N/A)
- [x] I have added documentation for all new/changed functionality (or N/A)

### CODE REVIEW

File: internal/cli/log_streams.go [hunk 1]
```diff
@@ -289,7 +289,11 @@ func (c *cli) logStreamPickerOptionsByType(desiredType logStreamType) pickerOpti
 			}
 		}
 		if len(options) == 0 {
-			return nil, fmt.Errorf("there are currently no log streams of type: %s", desiredType)
+			return nil, fmt.Errorf(
+				"there are currently no log streams of type: %q, use 'auth0 logs streams create %s' to create one",
+				desiredType,
+				desiredType,
+			)
```
Comment (sergiught) [inline internal/cli/log_streams.go:296]: We always give command hints in other places, so this just makes things consistent.

File: internal/cli/log_streams_datadog.go [hunk 2]
```diff
@@ -132,6 +132,16 @@ func updateLogStreamsDatadogCmd(cli *cli) *cobra.Command {
...
+			if oldLogStream.GetType() != string(logStreamTypeDatadog) {
+				return fmt.Errorf(
+					"the log stream with ID %q is of type %q instead of datadog, "+
+						"use 'auth0 logs streams update %s' to update it instead",
+					inputs.ID,
+					oldLogStream.GetType(),
+					oldLogStream.GetType(),
+				)
+			}
+
```
Comment (sergiught) [inline internal/cli/log_streams_datadog.go:144]: Here we prevent a panic down below when we type cast the sink property, this safeguards us against passing a real log stream ID but of different type. This is replicated in all the update commands.

File: internal/cli/log_streams_event_bridge.go [hunk 3]
```diff
@@ -97,7 +97,7 @@ func updateLogStreamsAmazonEventBridgeCmd(cli *cli) *cobra.Command {
 
 	cmd := &cobra.Command{
 		Use:   "eventbridge",
-		Args:  cobra.NoArgs,
+		Args:  cobra.MaximumNArgs(1),
```
Comment (sergiught) [inline internal/cli/log_streams_event_bridge.go:100]: This is a bug where we weren't accepting any log stream id for the update command. We fix this here and in other update commands down below.

**Review by Widcket — APPROVED** [review]
Good work here improving the test coverage, just pointed out a couple of non-blocking nits.

File: internal/cli/log_streams_event_grid.go [hunk 4]
```diff
@@ -139,6 +139,16 @@ func updateLogStreamsAzureEventGridCmd(cli *cli) *cobra.Command {
 				return fmt.Errorf("failed to read log stream with ID %s: %w", inputs.ID, err)
 			}
 
+			if oldLogStream.GetType() != string(logStreamTypeAzureEventGrid) {
+				return fmt.Errorf(
+					"the log stream with ID %q is of type %q instead of eventgrid, "+
```
[the remark this answers is not in the document]
Comment (sergiught) [reply internal/cli/log_streams_event_grid.go]: Fixed in [`129d72f` (#663)](https://github.com/auth0/auth0-cli/pull/663/commits/129d72f4b866c6e77cb6d45e176587849bc86290)

**Review by willvedd — APPROVED** [review]
Great work 👍

**Pull request merged**